charts/tekton/templates/eventlistener.yaml

```diff
@@ -0,0 +1,74 @@
+{{ if .Values.eventlistener.create }}
+
+apiVersion: triggers.tekton.dev/v1beta1
+kind: EventListener
+metadata:
+  name: {{ include "tekton.eventlistenerName" .Values.eventlistener.suffix }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
this should be "ci" namespace, so your values in chart
![image](https://github.com/user-attachments/assets/5595d3b0-6987-48e0-b1f7-684a0f704365)

should have
```
eventlistener:
  namespace: ci
```

to be set

Reply:
and only if not set - then use .Release.Namespace

the reason
when I debug stuff in ci namespace, I don't want to switch to a different NS or pass it via kubectl -n arg.

Reply:
![image](https://github.com/user-attachments/assets/419eeb84-7e6f-4ee2-8e09-281f57044822)

Reply:
done